### PR TITLE
Added section on ByChained

### PR DESCRIPTION
**Pull request by shbenzer**
### **User description**
Added a section on ByChained to the documentation

### Description
Split the By section into two subsections: By and ByChained
Added description of ByChained
Added code example to ByChained
Added to all translations

### Motivation and Context
Makes documentation more comprehensive
Provides code examples for features

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Documentation

___

### **Description**
- Introduced a new section `ByChained` in the documentation for multiple languages (English, Japanese, Portuguese, and Chinese).
- Provided an HTML example to demonstrate the use of `ByChained` for element location.
- Added a Java code example to illustrate how to implement `ByChained`.
- Enhanced the documentation to make it more comprehensive by explaining the chaining of locators.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>locators.en.md</strong><dd><code>Add `ByChained` section with example in English documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/locators.en.md

<li>Added a new section titled <code>ByChained</code>.<br> <li> Provided an HTML example to illustrate usage.<br> <li> Included a Java code example for <code>ByChained</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1911/files#diff-bad5c08d78f84e8bd7528d1a2a3181973375147d3f8bc5b9678cfce7dbe76aab">+44/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>locators.ja.md</strong><dd><code>Add `ByChained` section with example in Japanese documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/locators.ja.md

<li>Added a new section titled <code>ByChained</code>.<br> <li> Provided an HTML example to illustrate usage.<br> <li> Included a Java code example for <code>ByChained</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1911/files#diff-06ad92a757672227e965b2d837608b89da6bb02315528815d9e6fa2eac9114af">+42/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>locators.pt-br.md</strong><dd><code>Add `ByChained` section with example in Portuguese documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/locators.pt-br.md

<li>Added a new section titled <code>ByChained</code>.<br> <li> Provided an HTML example to illustrate usage.<br> <li> Included a Java code example for <code>ByChained</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1911/files#diff-d1f5d96e2cf4217d2473ebd28f060129cedf8bde199fc884493ffd01d205d994">+44/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>locators.zh-cn.md</strong><dd><code>Add `ByChained` section with example in Chinese documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/elements/locators.zh-cn.md

<li>Added a new section titled <code>ByChained</code>.<br> <li> Provided an HTML example to illustrate usage.<br> <li> Included a Java code example for <code>ByChained</code>.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1911/files#diff-275209ad35c57329488099cdbd775ba45e6386d21561836642e99c0b67721f10">+44/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by shbenzer** [comment]
This PR also covers #1287

**Comment by shbenzer** [comment]
If the response to this is positive, I will add a section on ByAll

File: website_and_docs/content/documentation/webdriver/elements/locators.en.md [hunk 1]
```diff
@@ -366,6 +368,48 @@ others it's as simple as setting a parameter in the FindElement function
...
+    </body>
+</html>
+```
+
+and you need to access the input of type text, instead of the input of type number. With ByChained you can access the desired input by chaining two Bys together, first to the div with `id="parentDiv"` and from that div to its child: `id="phoneNumber"`.
+
+{{< tabpane langEqualsHeader=true >}}
+{{< badge-examples >}}
+  {{< tab header="Java" >}}
+    import org.openqa.selenium.By;
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/elements/locators.en.md]: Is it possible to have a working example code and link it here?

Comment (shbenzer) [reply website_and_docs/content/documentation/webdriver/elements/locators.en.md]: Did it

File: examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java [hunk 2]
```diff
@@ -1,7 +1,31 @@
...
+
+    // Find element using ByChained
+    // utilizes html described on page
+    By example = new ByChained(By.id("parentDiv"), By.id("phoneNumber"));
+    driver.findElements(example);
+
 }
+
+
+/* ByChained examples utilizes the following HTML located in locators.md
```
Comment (harsha509) [inline examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java]: Apologies if my earlier message was unclear...!

You can use any of the examples from [this page](https://www.selenium.dev/selenium/web/), and I'd appreciate having a working example. 

I think the [login example](https://www.selenium.dev/selenium/web/login.html) should be sufficient for demonstrating the concept.

Comment (shbenzer) [reply examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java]: Ahhh, I see what you mean. I'll refactor

Comment (shbenzer) [reply examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java]: Just made a new push, let me know what you think

File: examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+
+        // Find username-field inside of login-form
+        By example = new ByChained(By.id("login-form"), By.id("username-field"));
+        WebElement username_input = driver.findElement(example);
+
+        //return placeholder text
+        String placeholder = username_input.getAttribute("placeholder");
+        return placeholder;
+    }
+
```
Comment (harsha509) [inline examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java]: we can remove extra space!

File: examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java [hunk 4]
```diff
@@ -1,7 +1,27 @@
...
 import dev.selenium.BaseTest;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 
 public class LocatorsTest extends BaseTest {
 
-}
+    public String ByChainedTest() 
+    {
```
Comment (harsha509) [inline examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java]: public String ByChainedTest() {

File: website_and_docs/content/documentation/webdriver/elements/locators.en.md [hunk 5]
```diff
@@ -366,6 +368,32 @@ others it's as simple as setting a parameter in the FindElement function
...
 {{< /tabpane >}} 
 
+### ByChained
+
+The `ByChained` class enables you to _chain_ two By locators together. For example, instead of having to locate a parent element, and then a child element of that parent, you can instead combine those two `FindElement` functions into one.
+
+{{< tabpane langEqualsHeader=true >}}
+{{< badge-examples >}}
+  {{< tab header="Java" >}}
+    {{< gh-codeblock path="examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java#11-25" >}}
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/elements/locators.en.md]: I believe 
L18-20 should be enough!

Comment (shbenzer) [reply website_and_docs/content/documentation/webdriver/elements/locators.en.md]: done

File: examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java (hunk 3 above)
Comment (shbenzer) [reply examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java]: done

**Review by harsha509 — APPROVED** [review]
Thank you @shbenzer!

**Pull request merged**